tests/test_mysql.py

```diff
@@ -1,52 +1,149 @@
-import unittest
-from tests.common import load_check
+# stdlib
 from nose.plugins.attrib import attr
-import time
+
+# project
+from tests.common import AgentCheckTest
+
 
 @attr(requires='mysql')
-class TestMySql(unittest.TestCase):
+class TestMySql(AgentCheckTest):
+    CHECK_NAME = 'mysql'
+
+    METRIC_TAGS = ['tag1', 'tag2']
+    SC_TAGS = ['host:localhost', 'port:0']
+
+    MYSQL_CONFIG = {
+        'init_config': {},
+        'instances': [{
+            'server': 'localhost',
+            'user': 'dog',
+            'pass': 'dog',
+            'options': {'replication': True},
+            'tags': METRIC_TAGS
+        }]
+    }
+
+    FAILURE_CONNECTION = {
+        'init_config': {},
+        'instances': [{
+            'server': 'localhost',
+            'user': 'unknown',
+            'pass': 'dog',
+        }]
+    }
+
+    # Available by default on MySQL > 5.5
+    INNODB_METRICS = [
+        "mysql.innodb.buffer_pool_free",
+        "mysql.innodb.buffer_pool_used",
+        "mysql.innodb.buffer_pool_total",
+        "mysql.innodb.buffer_pool_utilization"
+    ]
+
+    REPLICATION_METRICS = [
+        "mysql.replication.slave_running"
+    ]
+
+    KEY_CACHE = [
+        "mysql.performance.key_cache_utilization"
+    ]
+
+    # Available on Linux
+    SYSTEM_METRICS = [
+        "mysql.performance.user_time",
+        "mysql.performance.kernel_time"
```

Comment:
Ultra nitpick: but please use single quotes everywhere. Or or the opposite if you prefer to...